src/const.ts

```diff
@@ -21,6 +23,7 @@ export interface Manifest {
   new_install_prompt_erase?: boolean;
   /* Time to wait to detect Improv Wi-Fi. Set to 0 to disable. */
   new_install_improv_wait_time?: number;
+  serialPortFilter: SerialPortFilter[];
```

Comment:
Just make this `serialPortRequestOptions` so we never have to add any other option to the manifest that can be passed in.